### PR TITLE
minor: Disable removed clusters-only send empty config

### PR DESCRIPTION
**Pull request by c1freitas**
We don't want to send the SHA or deployId as empty strings, but we should be sending the config. This sets the config to an empty object for now.  

Will also use this PR to fix https://github.com/InVisionApp/elroy/issues/220

### CODE REVIEW

**Comment by c1freitas** [comment]
This incorporates https://github.com/InVisionApp/kit-deploymentizer/pull/55

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   "main": "./src/index.js",
   "bin": "./src/deploymentizer",
   "scripts": {
-    "test-unit": "mocha --recursive test/unit",
+    "test-unit": "mocha --recursive test/unit --timeout 15000",
```
Comment (chesleybrown) [inline package.json]: ❓

File: test/unit/lib/elroy-sync.spec.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
+"use strict";
```
Comment (chesleybrown) [inline test/unit/lib/elroy-sync.spec.js]: Unit tests 👏

File: package.json (hunk 1 above)
Comment (c1freitas) [reply package.json]: 🙈

**Pull request merged**